presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2591,6 +2594,40 @@ boolean isIncreasingSequence(List<Integer> fileNames)
         return true;
     }
 
+    @DataProvider(name = "reservedWordsTestDataProvider")
+    public static Object[][] reservedWordsTestDataProvider()
+    {
+        return ReservedIdentifiers.reservedIdentifiers().stream()
+                .map(a -> {
+                    String reservedIdentifier = a.toLowerCase(Locale.ENGLISH);
+                    return new Object[] {"test_show_create_reserved_" + reservedIdentifier, reservedIdentifier};
+                })
+                .toArray(size -> new Object[size][2]);
+    }
+
+    @Test(dataProvider = "reservedWordsTestDataProvider")
+    public void testShowCreateWithReservedIdentifier(String tableName, String reservedIdentifier)
+    {
+        String createTableSql = format("" +
+                        "CREATE TABLE %s.%s.%s (\n" +
+                        "   \"" + reservedIdentifier + "\" bigint,\n" +
+                        "   c2 double,\n" +
+                        "   \"c 3\" varchar,\n" +
```

Comment:
It is nice to test not only reserved keywords, but also names with special characters like space and single quote. However, these are already tested in testShowCreateTable. It is not clear what would be the value of duplicating these tests. Also, it seems that this new test runs 69 or so times (as many as there are reserved keywords. I wonder how long it takes to run these tests and whether the potentially long runtime is worth it? Why not just hard-code a few columns using some of the reserved keywords? That would also make tests easier to read.